packages/perseus-editor/src/editor.tsx

```diff
@@ -31,7 +31,7 @@ import type {
     PerseusWidgetsMap,
 } from "@khanacademy/perseus";
 
-// like [[snowman input-number 1]]
+// like [[snowman numeric-input 1]]
 const widgetPlaceholder = "[[\u2603 {id}]]";
```

Comment:
Not a critical change, but we might as well have our example use a widget name that will still exist after this work.